packages/client/modules/userDashboard/components/OrganizationRow/OrganizationRow.tsx

```diff
@@ -139,9 +139,7 @@ const OrganizationRow = (props: Props) => {
               </StyledTagBlock>
             )}
           </RowInfoHeader>
-          <StyledRowInfoCopy>
-            {`${totalUsers} ${plural(totalUsers, 'User')} (${activeUserCount} Active)`}
-          </StyledRowInfoCopy>
+          <StyledRowInfoCopy>{`${totalUsers} ${plural(totalUsers, 'User')}`}</StyledRowInfoCopy>
```

Comment:
-1 we're removing `active` for all users here. It's still relevant for Team users, so I think it should only be hidden for Enterprise users